accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml

```diff
@@ -14,5 +14,5 @@
 
 <workspaceFilter version="1.0">
     <filter root="/oak:index/repACL" type="cleanup"/><!-- legacy index node name, used prior version 2.7.2  -->
-    <filter root="/oak:index/repACL-custom-1"/>
```

Comment:
we need to remove the old one, please keep the old filter rule with type="cleanup"